vnext/Microsoft.ReactNative/Views/FlyoutViewManager.cpp

```diff
@@ -384,7 +384,16 @@ void FlyoutShadowNode::SetTargetFrameworkElement() {
       }
     }
   } else {
-    m_targetElement = xaml::Window::Current().Content().as<xaml::FrameworkElement>();
+    if (IsXamlIsland()) {
+      // // XamlRoot added in 19H1
+      if (Is19H1OrHigher()) {
+        if (auto xamlRoot = React::XamlUIService::GetXamlRoot(GetViewManager()->GetReactContext().Properties())) {
+          m_targetElement = xamlRoot.Content().try_as<xaml::FrameworkElement>();
```

Comment:
nit, can probably be a direct `as` instead of `try_as` here (it will crash if it isn't a FE, but it's not like letting it run further will help anyway :))

Reply:
```suggestion
          m_targetElement = xamlRoot.Content().as<xaml::FrameworkElement>();
```